src/components/FeatureChecking/util.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+import get from 'lodash/get';
+import * as types from './flowTypes';
+
+function isFeatureEnabled(features: types.FeatureConfig, featureName: string) {
+    return !!get(features, `${featureName}`, false);
```

Comment:
Is the template string necessary here? It looks like `featureName` is required to be a string as-is.

Reply:
Good point, that's residual from this format: `${featureName}.enabled` (requiring an object)